src/WholeBodyStateDisplay.cpp

```diff
@@ -289,12 +301,22 @@ void WholeBodyStateDisplay::loadRobotModel() {
   if (!descr.initString(robot_model_)) {
     clearRobotModel();
     setStatus(StatusProperty::Error, "URDF", "Failed to parse URDF model");
+    ROS_ERROR("Failed to parse URDF model");
     return;
   }
 
   // Initializing the dynamics from the URDF model
-  pinocchio::urdf::buildModelFromXML(robot_model_,
-                                     pinocchio::JointModelFreeFlyer(), model_);
+  try {
+    pinocchio::urdf::buildModelFromXML(
+        robot_model_, pinocchio::JointModelFreeFlyer(), model_);
+  } catch (const std::invalid_argument &e) {
+    std::string error_msg = "Failed to instantiate model: ";
+    error_msg += e.what();
+    setStatus(StatusProperty::Error, "Pinocchio-URDFParser",
+              QString::fromStdString(error_msg));
+    ROS_ERROR_STREAM(error_msg);
```

Comment:
and same question here

Reply:
This error was too long to be read properly otherwise I believe